crates/router/src/core/webhooks/incoming.rs

```diff
@@ -1646,21 +1623,20 @@ async fn disputes_incoming_webhook_flow(
             connector.id(),
         )
         .await?;
-        let disputes_response = Box::new(dispute_object.clone().foreign_into());
-        let event_type: enums::EventType = dispute_object.dispute_status.foreign_into();
 
-        Box::pin(super::create_event_and_trigger_outgoing_webhook(
+        let event_type: enums::EventType = dispute_object.dispute_status.foreign_into();
+        Box::pin(super::add_bulk_outgoing_webhook_task_to_process_tracker(
             state,
-            merchant_context,
-            business_profile,
+            &business_profile,
+            &dispute_object.dispute_id.clone(),
             event_type,
-            enums::EventClass::Disputes,
-            dispute_object.dispute_id.clone(),
-            enums::EventObjectType::DisputeDetails,
-            api::OutgoingWebhookContent::DisputeDetails(disputes_response),
+            diesel_models::enums::EventClass::Disputes,
+            diesel_models::enums::EventObjectType::DisputeDetails,
             Some(dispute_object.created_at),
         ))
-        .await?;
+        .await
+        .map_err(|e| e.change_context(errors::ApiErrorResponse::WebhookProcessingFailure))?;
```

Comment:
Same comments as before, about the `enums::` qualification and the unnecessary `map_err()`.